src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,480 @@
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="time-ranges"
+    title="3. Use Time Ranges"
+  >
+<SideBySide>
+  <Side>
+    Every piece of data in NRDB has a `timestamp`, and every query operates on a subset of data within a time range. If you don't provide a time range in your NRQL query, it will return the last 60 minutes by default as we saw in the previous lesson. But what if we want to see data from a different time range? How do we control the window of data we see? The answer to these questions is very straightforward: use `SINCE` and `UNTIL` keywords to provide a beginning and end to your query's time range.
+```sql 
+SELECT average(duration) 
+FROM Transaction 
+SINCE 1 day ago
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 day ago query"
+  alt="A screenshot of a query selecting the average duration since 1 day ago"
+  src={queriesnrqltutorial9}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    We can also specify time ranges relative to when we make our queries by using keywords like `day`, `week`, `hour`, `minute`, or their plural equivalents. We can also use logical expressions like `SINCE` today or `SINCE` this week.
+```sql
+SELECT average(duration) 
+FROM Transaction 
+SINCE this week
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 week ago"
+  alt="A screenshot of a query selecting the average duration since 1 week ago"
+  src={queriesnrqltutorial10}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    Here is another query that demonstrate relative time ranges. Note that when specifying a relative time, you need to include the keyword `ago`.
+```sql
+SELECT average(duration) 
+FROM Transaction 
+SINCE 1 week ago 
+UNTIL 2 days ago
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 week ago until 2 days ago"
+  alt="A screenshot of a query selecting the average duration since 1 week ago until 2 days ago"
+  src={queriesnrqltutorial11}
+/>
+  </Side>
+</SideBySide>
+Now we're getting even more control of our data. By adding a time frame to a query, we're define exactly which window of data we want to see. These controls let us zero in on the data we want by defining where the data reporting should start and end. In our examples, we saw that we can easily choose the window and use common terms like `week`, `day`, `hour`, `minute`, and their plurals. You can even use `month` and `year` if you're storing that much data in New Relic. Let's go to the next step: querying a time series.
```

Comment:
@jeff-colucci looks like a typo. Should be "...we're defining exactly...."

Reply:
Thank you!